### PR TITLE
CNV#36162: RN - NEW - Search option in VM configuration tab

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/CNV-36162

Link to docs preview:
https://71699--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-web

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -109,6 +109,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-36165: VM disks with persistent reservation
 
 //CNV-36162: Search option in VM configuration tab
+* A virtual machine's configuration section now has a search field.
```
Comment (gouyang) [inline virt/release_notes/virt-4-15-release-notes.adoc:112]: The feature is about adding a search field in the VM configuration tab, so user can find the configure filed easily by searching, could you rephase it a bit? :)

**Review by skopacz1 — APPROVED** [review]
LGTM

**Pull request merged**